src/pdc-api.ts

```diff
@@ -7,252 +7,245 @@ const logger = getLogger('pdc-api');
-     * state may change from render to render, such changes are not relevant:
-     * a change in the way we make a request should not trigger an API request
-     *
-     * params is a dependency, but as an object - and often a newly-created
-     * object - depending on it directly causes spurious renders; instead, use
-     * its string value, which should be stable
-     */
-  }, [path, params.toString()]);
-
-  useEffect(() => {
-    fetchData();
-  }, [fetchData]);
-
-  return [
-    response,
-    fetchData,
-  ];
+	const { fetch } = useOidcFetch();
+	const [response, setResponse] = useState(null);
+
+	const fetchData = useCallback(() => {
+		setResponse(null);
+		const url = new URL(path, API_URL);
+		url.search = params.toString();
+		fetch(url)
+			.then(throwIfResponseIsNotOk)
+			.then((res) => res.json())
+			.then(setResponse)
+			.catch((e) => logError(e, path, params));
+
+		/* eslint-disable-next-line react-hooks/exhaustive-deps --
+		 *
+		 * fetch should not be a dependency, because although it or its internal
+		 * state may change from render to render, such changes are not relevant:
+		 * a change in the way we make a request should not trigger an API request
+		 *
+		 * params is a dependency, but as an object - and often a newly-created
+		 * object - depending on it directly causes spurious renders; instead, use
+		 * its string value, which should be stable
+		 */
+	}, [path, params.toString()]);
+
+	useEffect(() => {
+		fetchData();
+	}, [fetchData]);
+
+	return [response, fetchData];
 };
 
 const usePdcCallbackApi = <T>(
-  path: string,
-): (options: RequestInit) => Promise<T> => {
-  const { fetch } = useOidcFetch();
-
-  return useCallback((options: RequestInit) => {
-    const url = new URL(path, API_URL);
-    return fetch(url, options)
-      .then(throwIfResponseIsNotOk)
-      .then((res) => res.json())
-      .catch((e) => logError(e, path, options));
-
-    /* eslint-disable-next-line react-hooks/exhaustive-deps --
-     *
-     * fetch should not be a dependency, because although it or its internal
-     * state may change from render to render, such changes are not relevant:
-     * a change in the way we make a request should not trigger an API request
-     */
-  }, [path]);
+	path: string,
+): ((options: RequestInit) => Promise<T>) => {
+	const { fetch } = useOidcFetch();
+
+	return useCallback(
+		(options: RequestInit) => {
+			const url = new URL(path, API_URL);
+			return fetch(url, options)
+				.then(throwIfResponseIsNotOk)
+				.then((res) => res.json())
+				.catch((e) => logError(e, path, options));
+
+			/* eslint-disable-next-line react-hooks/exhaustive-deps --
+			 *
+			 * fetch should not be a dependency, because although it or its internal
+			 * state may change from render to render, such changes are not relevant:
+			 * a change in the way we make a request should not trigger an API request
+			 */
+		},
+		[path],
```

Comment:
After Prettifying this section, the `eslint-disable-next-line` command is now _two_ lines away from the target line (`[path],`), which means ESLint complains again.

Can you edit this commit to move the command to just above the `[path],` line? The entire return should look like:

```ts
	return useCallback(
		(options: RequestInit) => {
			const url = new URL(path, API_URL);
			return fetch(url, options)
				.then(throwIfResponseIsNotOk)
				.then((res) => res.json())
				.catch((e) => logError(e, path, options));
		},
		/* eslint-disable-next-line react-hooks/exhaustive-deps --
		 *
		 * fetch should not be a dependency, because although it or its internal
		 * state may change from render to render, such changes are not relevant:
		 * a change in the way we make a request should not trigger an API request
		 */
		[path],
	);
```

You'll once again have to update the following commit with the new hash, sorry!

I'd recommend doing an `npm run lint` before actually committing this to be sure it's all clean. 🙏